apps/minifront/src/state/send/index.ts

```diff
@@ -19,34 +19,9 @@ import {
 import { getAddress, getAddressIndex } from '@penumbra-zone/getters/address-view';
 import { toBaseUnit } from '@penumbra-zone/types/lo-hi';
 import { isAddress } from '@penumbra-zone/bech32m/penumbra';
-import { ZQueryState, createZQuery } from '@penumbra-zone/zquery';
-import { getTransferableBalancesResponses } from '../components/send/helpers';
-
-export const { transferableBalancesResponses, useTransferableBalancesResponses } = createZQuery({
```

Comment:
We now use `shared.balancesResponses` for the Send form, plus a `transferableBalancesResponsesSelector` to select only transferable balances responses.

Then we use middleware (farther down in this file) to check for when `shared.balancesResponses` has become populated, and update `state.send.selection` if it's not already set.